src/databases.py

```diff
@@ -49,7 +48,7 @@ def __init__(self, charm: CharmBase):
             for require in metadata["requires"].values()
             if require["interface"] in FLASK_SUPPORTED_DB_INTERFACES
         )
-
+        # automatically create database relation requirers to manage database relations
```

Comment:
Perhaps expand this a bit more to indicate why a dictionary is used?

Reply:
Added, thanks!